assets/js/base/hooks/cart/use-store-cart.ts

```diff
@@ -21,20 +31,49 @@ const defaultShippingAddress = {
 	country: '',
 };
 
-const defaultBillingAddress = {
+const defaultBillingAddress: CartResponseBillingAddress = {
 	...defaultShippingAddress,
 	email: '',
 	phone: '',
 };
 
-const decodeValues = ( object ) =>
-	mapValues( object, ( value ) => decodeEntities( value ) );
+const defaultCartTotals: CartResponseTotals = {
+	total_items: '',
+	total_items_tax: '',
+	total_fees: '',
+	total_fees_tax: '',
+	total_discount: '',
+	total_discount_tax: '',
+	total_shipping: '',
+	total_shipping_tax: '',
+	total_price: '',
+	total_tax: '',
+	tax_lines: [],
+	currency_code: '',
+	currency_symbol: '',
+	currency_minor_unit: 2,
+	currency_decimal_separator: '',
+	currency_thousand_separator: '',
+	currency_prefix: '',
+	currency_suffix: '',
+};
+
+const decodeValues = (
+	object: Record< string, unknown >
+): Record< string, unknown > => {
+	return Object.fromEntries(
+		Object.entries( object ).map( ( [ key, value ] ) => [
+			key,
+			decodeEntities( value ),
+		] )
+	);
+};
```

Comment:
Moved from lodash to `fromEntries` and `entries` cc @Aljullu is using `fromEntries` still a problem?

Reply:
@senadir yes it is. We have to use a polyfill. See `deriveSelectedShippingRates`

Reply:
Fixed with a new `fromEntriesPolyfill` function that's used in both places.